### DISCUSSION TITLE
EPG deixou de funcionar no KODI

### DISCUSSION THREAD

**Discussion by marieiro** [LITUATUI/M3UPT#41] [General]
Olá,

Muito obrigado por este projeto. Tenho estado a usar o M3UPT a algum tempo no KODI. A umas semanas atraz no entanto o EPG deixou de funcionar. Noutras apps da android não.  Houve alguma mudança no format do XML?

**Comment by LITUATUI** [comment]
O problema ainda persiste? Talvez estejas a usar uma versão antiga que não suporta EPG em format XZ.
